account_bank_statement_import_txt_xlsx/__manifest__.py

```diff
@@ -1,28 +1,35 @@
-# Copyright 2014-2017 Akretion (http://www.akretion.com).
-# @author Alexis de Lattre <alexis.delattre@akretion.com>
-# @author Sébastien BEAU <sebastien.beau@akretion.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
```

Comment:
The headers From Akretion were just copied from a template.

Please change to Copyright 2019 ForgeFlow, S.L.

as we have changed the company name.

Reply:
Done. It would be great if you could review the PR as well :)